node/v2/index.js

```diff
@@ -27,6 +27,7 @@ module.exports.Types = Types;
 
 var RetryFlags = require('../retry-flags.js');
 var Frame = require('./frame');
+var LazyFrame = require('./frame');
```

Comment:
in other news... with the newly lazy frame being added but dead path, I'm now amazed that there was any performance difference from master